tests/test_envs.py

```diff
@@ -1,32 +1,29 @@
+import warnings
+
 import gymnasium as gym
 import numpy as np
+from matplotlib import pyplot as plt
 
 import neurogym as ngym
 from neurogym.utils.data import Dataset
 
 try:
     import psychopy  # noqa: F401
 
-    _have_psychopy = True
+    _have_psychopy = True  # FIXME should psychopy be always tested, to ensure CI doesn't fail?
 except ImportError:
     _have_psychopy = False
 
 ENVS = ngym.all_envs(psychopy=_have_psychopy, contrib=True, collections=True)
 # Envs without psychopy, TODO: check if contrib or collections include psychopy
 ENVS_NOPSYCHOPY = ngym.all_envs(psychopy=False, contrib=True, collections=True)
+SEED = 0
 
 
-def make_env(env, **kwargs):
-    # use ngym.make and not gym.make to disable env_checker
-    return ngym.make(env, **kwargs)
-
-
-def test_run(env=None, num_steps=100, verbose=False, **kwargs):
+def _test_run(env, num_steps=100, verbose=False):
     """Test if one environment can at least be run."""
-    if env is None:
-        env = ngym.all_envs()[0]
     if isinstance(env, str):
-        env = make_env(env, **kwargs)
+        env = ngym.make(env)
```

Comment:
because it is empty. No actual kwargs are sent to the function in most cases. The `make_env` function, was an internal only function that literally called `ngym.make` and did nothing else, so I removed that function and only sent kwargs in case there were any to send.